tests/handlers/test_profile.py

```diff
@@ -44,8 +44,6 @@ def register_query_handler(query_type, handler):
 
         hs = yield setup_test_homeserver(
```

Comment:
I don't think we need to `yield` here as you mentioned above.

Reply:
true, but that can be fixed another time :innocent: